paf-mvp-core-js/src/crypto/key-store.ts

```diff
@@ -1,22 +1,22 @@
-import { fromIdentityResponse, KeyInfo } from './identity';
+import { fromIdentityResponse, PublicKeyInfo } from './identity';
 import { PublicKey, publicKeyFromString } from './keys';
 import { GetIdentityRequestBuilder } from '@core/model/identity-request-builder';
 import { GetIdentityResponse, Timestamp } from '@core/model/generated-model';
 import { getTimeStampInSec } from '@core/timestamp';
 import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
 
-export type PublicKeyInfo = KeyInfo & { publicKeyObj: PublicKey };
+export type PublicKeyWithObject = PublicKeyInfo & { publicKeyObj: PublicKey };
```

Comment:
`PublicKeyWithConfiguration` ? `IdentityConfiguration` ? `DomainIdentityInfo` ?

'WithObject' is not really descriptive.